client/src/Components/RoutedHeader.js

```diff
@@ -7,8 +7,8 @@ import RingdownIcon from './Icons/Ringdown';
 import HospitalIcon from './Icons/Hospital';
 
 const tabs = [
-  { label: 'Ringdown', Icon: RingdownIcon },
-  { label: 'Hospital Info', Icon: HospitalIcon },
+  { label: 'Ringdown', Icon: RingdownIcon, key: 'ringdownForm' },
+  { label: 'Hospital Info', Icon: HospitalIcon, key: 'hospitalStatuses' },
```

Comment:
It might make sense to use `id` instead of `key`, since React has its own use for that prop, and it avoids `tabKey` below, which can mean something else.